packages/overlay/src/active-overlay.css

```diff
@@ -35,7 +35,7 @@ governing permissions and limitations under the License.
 }
 
 :host {
-    z-index: 2;
+    z-index: 1000;
```

Comment:
@Westbrook This fixes the z issue. I was not sure how to set this for overlays. It looks like spectrum-css uses 100 for side navs, so we have to be higher than that...

Reply:
I think this is great. I'm gonna make a ticket to review this at a later date as it should probably be powered by CSS custom properties, but should be backed by some strong agreement of what it actually empowers... https://github.com/adobe/spectrum-web-components/issues/485

Reply:
Isn't this sort of problem why we made it possible to send overlays to specific places through the event system?

i.e. you would have an 'overlay container' div at the document root level, this would have a z-index defined for the overlay plane, and we'd have all our overlay content transferred to this root when popping up?

Likewise I could see us setting up several different containers for different z-index destined content, i.e. tooltips are always at the very top most plane, dialogs are a plane below etc.

Reply:
Ah actually taking a look at our current overlay-stack behavior we don't currently allow this.

I think it'd be really great if the `sp-overlay-query` event could carry with it a `overlay-type` property and allow handlers of the event to set a `overlay-root` property on the event payload. This would then allow us to carry a 'tooltip' value for the `overlay-type` property for tooltip overlays, and for consumers to add a new root for the overlays that are 'tooltips' that can then have specific z-indexing. This allows us to explicitly control at the document level both where overlays are destined to be hoisted to, and the z-index ordering at the document level. This would allow more natural use of z-index ordering I think as well?

Reply:
Agree that now that we have popper available we should do a review of the broader functionality like this that we want to power from this new foundation. I've added #487 with a quote of the above to discuss as I'm not sure it's specifically related to this PR.